components/log-viewer-webui/client/package.json

```
@@ -21,13 +21,20 @@
     "antd": "^5.24.5",
     "axios": "^1.7.9",
     "react": "^19.0.0",
-    "react-dom": "^19.0.0"
+    "react-dom": "^19.0.0",
+    "socket.io-client": "^4.8.1"
   },
   "devDependencies": {
+    "@eslint/js": "^9.19.0",
     "@types/react": "^19.0.10",
     "@types/react-dom": "^19.0.4",
     "@vitejs/plugin-react": "^4.3.4",
+    "eslint": "^9.19.0",
     "eslint-config-yscope": "latest",
+    "eslint-plugin-react": "^7.37.4",
+    "eslint-plugin-react-hooks": "^5.0.0",
+    "eslint-plugin-react-refresh": "^0.4.18",
+    "globals": "^15.14.0",
```

We don't have to explicitly state those eslint dependencies because `eslint-config-yscope` lists those as "peer" dependencies, so `npm install` will install those even if not explicitly specified.
```suggestion
    "@types/react": "^19.0.10",
    "@types/react-dom": "^19.0.4",
    "@vitejs/plugin-react": "^4.3.4",
    "eslint-config-yscope": "latest"
```